pkg/graph/resolvers/plan_discussion.go

```diff
@@ -454,63 +454,73 @@ func CreateDiscussionReply(
 		if err != nil {
 			return reply, err
 		}
+
 		// Create Activity and notifications in the DB
 		_, notificationErr := notifications.ActivityTaggedInDiscussionReplyCreate(ctx, tx, principal.Account().ID, discussion.ID, reply.ID, reply.Content, loaders.UserNotificationPreferencesGetByUserID)
 		if notificationErr != nil {
 			return nil, fmt.Errorf("unable to generate notifications, %w", notificationErr)
 		}
-		go func() {
-
-			replyUser := principal.Account()
-			commonName := replyUser.CommonName
 
-			if err != nil {
-				logger.Error("error sending discussion reply emails. Unable to retrieve modelPlan",
-					zap.String("replyID", reply.ID.String()),
-					zap.Error(err))
-			}
+		pref, err := UserNotificationPreferencesGetByUserID(ctx, principal.Account().ID)
```

Comment:
I think this is getting preferences for the wrong user, it should be getting the preferences of the user who created the discussion, not the preferences of the user who made the reply. The discussion creator should get the notification